tracer/src/Datadog.Trace/Propagators/W3CTraceContextPropagator.cs

```diff
@@ -385,6 +400,10 @@ internal static W3CTraceState ParseTraceState(string header)
                     {
                         origin = value.ToString();
                     }
+                    else if (name.Equals(TraceStateLastParentKey, StringComparison.Ordinal))
+                    {
+                        lastParent = value.ToString();
+                    }
```

Comment:
Not for this PR, but we might want to consider unifying this code to remove duplication now we have the vendored span